qos-monitor/src/main/java/eu/arrowhead/core/qos/service/ping/monitor/PingEventBufferElement.java

```diff
@@ -0,0 +1,54 @@
+package eu.arrowhead.core.qos.service.ping.monitor;
+
+import java.io.Serializable;
+import java.util.UUID;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.util.Assert;
+
+import eu.arrowhead.core.qos.QosMonitorConstants;
+import eu.arrowhead.core.qos.dto.event.monitoringevents.MeasurementMonitoringEvent;
+
+public class PingEventBufferElement implements Serializable {
+
+	//=================================================================================================
+	// members
+
+	//-------------------------------------------------------------------------------------------------
+
+	private static final long serialVersionUID = -4236505546657946368L;
+
+	private final UUID id;
+	private final long createdAt;
+
+	private final MeasurementMonitoringEvent eventList[];
```

Comment:
if something is an array, then the array signs [] are part of the type. What you write here is correct syntax, but I prefer the
`MeasurementMonitoringEvent[] eventList` 
Also, if this is an arry why call it a list?

Reply:
fixed